backend/src/main/java/com/alibaba/higress/console/controller/DomainsController.java

```diff
@@ -56,6 +56,9 @@ public ResponseEntity<PaginatedResponse<Domain>> list(CommonPageQuery query) {
 
     @PostMapping
     public ResponseEntity<Response<Domain>> add(@RequestBody Domain domain) {
+        if (!domain.valid()) {
+            throw new ValidationException("Domain is invalid.");
```

Comment:
Is it possible to let caller know which part is invalid and why?

Reply:
If we want to know the reason. we need the `valid()` method return with some message